packages/components/addon/components/hds/dropdown/list-item/copy-item.js

```diff
@@ -38,14 +38,16 @@ export default class HdsDropdownListItemCopyItemComponent extends Component {
 
   @action
   async copyCode() {
-    // https://developer.mozilla.org/en-US/docs/Web/API/Clipboard
-    await navigator.clipboard.writeText(this.args.text);
-    const result = await navigator.clipboard.readText();
+    let copyText = await this.args.text;
+    // https://codepen.io/chrisdavidmills/full/gzYjag/
+    // execCommand is supposedly deprecated but it's also what Firefox says to do in this situation so we'll try it
+    // also execCommand is synchronous so we can't await it
+    document.execCommand('copy', false, copyText || '');
 
-    if (result === this.args.text) {
+    if (copyText === this.args.text) {
       this.isSuccess = true;
-      // console.log(`result is ${result}`);
-
+      // if you un-comment the next line you'll see what you expect in the browser when you visit the showcase for this component.
+      // console.log(`success: ${this.isSuccess}; copied text: ${copyText}`);
```

Comment:
[suggestion] i generally try to not to leave behind debugging statements, so I think this and the line above could be removed